plugin/example-spec.js

```diff
@@ -0,0 +1,31 @@
+/* jshint jasmine: true */
```

Comment:
You can quickly run this test spec through vscode using this launch configuration:

```
{
    // Use IntelliSense to learn about possible Node.js debug attributes.
    // Hover to view descriptions of existing attributes.
    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
    "version": "0.2.0",
    "configurations": [
        {
            "type": "chrome",
            "request": "launch",
            "name": "Test Spec",
            "runtimeExecutable": "G:/stingray/build/binaries/editor/stingray_editor.exe",
            "runtimeArgs": [
                "--test",
                "${file}"
            ],
            "timeout": 10000,
            "url": "file:///G:/stingray/editor/core/boot.html?*single-test-spec.js*",
            "pathMapping": {
                "${workspaceRoot}": "file:///${workspaceRoot}/"
            },
            "port": 9088
        }
    ]
}
```

This configuration will start Stingray editor and run the test spec.

The generated command line will be:
> G:/stingray/build/binaries/editor/stingray_editor.exe --remote-debugging-port=9088 --no-first-run --no-default-browser-check --test g:\stingray-plugin\plugin\example-spec.js file:///G:/stingray/editor/core/boot.html?*single-test-spec.js*